.editorconfig

```diff
@@ -13,6 +13,10 @@ indent_style = space
 
 trim_trailing_whitespace = true
 
+[*.rst]
+# Three-space indentation
+indent_size = 3
```

Comment:
I was wondering why 3, and then I found
https://devguide.python.org/documentation/style-guide/
> All reST files use an indentation of 3 spaces

Reply:
Yeah, it's a bit weird but then so is RST :)

https://stackoverflow.com/q/48311159/724176 says there's no "official" requirement.

Another random Google result:

> In reST, the indentation of a block of lines is often important. The exact number of spaces, which are used to indent a block of text, does not matter. But what does matter, is that all lines of the block are indented with exactly the same number of spaces.
>
> We use the convention of 4 spaces per indenting level (see [Whitespace and indentation](https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/GeneralConventions/CodingGuidelines.html#cgl-indenting)).

https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/WritingReST/BasicRestSyntax.html

The installation page uses 3 spaces. I did a cursory search of the docs and `git grep "  \.\." docs` suggests we're already using 3-space indents, but happy to remove this config if you prefer.

On the other hand, if we are going to use 3-space, this helps the IDE know how many to add when tabbing.

Reply:
Opening up the files inside docs/handbook, I don't think we (at least, not consistently) using 3 spaces at the moment. I think that the mix of 3 spaces for RST and 4 for Python indentation is weird.

But given that there's already a variety of style rules that we have to follow when writing documentation, I'm not overly concerned about adding another, and I think that specifying a value is helpful.

Reply:
Shall we prefer 4 over 3? I'm not too bothered which, and let's not bother mass-reindenting the docs.

Reply:
4!

Reply:
Updated to 4! Let's do a squash merge on this PR to keep the Git history a little neater.